src/providers/data_provider_fo.c

```diff
@@ -675,6 +683,12 @@ errno_t be_resolve_server_process(struct tevent_req *subreq,
         DEBUG(SSSDBG_FUNC_DATA, "Found address for server %s: [%s] TTL %d\n",
               fo_get_server_str_name(state->srv), ipaddr,
               srvaddr->addr_list[0]->ttl);
+    } else {
+        DEBUG(SSSDBG_CRIT_FAILURE, "Missing server name.\n");
```

Comment:
Idea is that this ^^ line triggers a backtrace and then following lines are printed with default log level cleanly (without backtrace)?

Reply:
Hi,

yes, but mainly I wanted a message even if the default debug level was lowered.

bye,
Sumit